File: Programmers/[12900] 2 x n 타일링/Johoseong/12900.py
def solution(n):
    a, b = 1, 2
    if n == 1: return 1
    if n == 2: return 2

    for _ in range(3, n + 1):
        c = a + b
        a, b = b, c
        # b = c
    return c % 1000000007

File: Programmers/[12900] 2 x n 타일링/KingDonggyu/12900.js
function solution(n) {
  const fibo = [1, 1, 2, 3, 5, 8];
  for (let i = 6; i <= n; i++) {
    fibo[i] = (fibo[i - 1] + fibo[i - 2]) % 1000000007;
  }
  return fibo[n];
}
